test/geos_capi/analysis_test.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+
+module GeosCapi
+  class AnalysisTest < Minitest::Test
+    def setup
+      skip "Needs GEOS." unless RGeo::Geos.capi_supported?
+    end
```

Comment:
Other c_api tests add the `if RGeo::Geos.capi_supported?` statement after the closure to the test class. I like this approach, so at some point, we could change all of them to this format. We can save that for another PR though.

Reply:
Yeah I saw the `if RGeo::Geos.capi_supported?` and the WARNING, but this is hard to see IMHO. I'm all for changing that. I won't be able to work on it until next week. If you have time and you want to add a commit to this PR I'd be OK. Or I'll open an issue and we'll add it to our low priority queue :)